packages/suite/src/views/wallet/send/components/Fees/index.tsx

```diff
@@ -10,28 +10,26 @@ import CustomFee from './components/CustomFee';
 import { useSendFormContext } from '@wallet-hooks';
 
 const StyledCard = styled(Card)`
-    display: flex;
-    justify-items: space-between;
     margin-bottom: 25px;
     padding: 32px 42px;
-
-    @media all and (max-width: ${variables.SCREEN_SIZE.SM}) {
```

Comment:
Some wrapping is still needed as the new layout won't work on smaller screens as the fee selector is taking too much space 
<img width="531" alt="Screenshot 2020-09-24 at 14 06 07" src="https://user-images.githubusercontent.com/6961901/94142785-1a0b6200-fe6f-11ea-9f8d-d89c1135abd7.png">


I would expect amounts move under the selector just like it used to be. Imo there is no need for grid as it should be easy job for flexbox.
<img width="592" alt="Screenshot 2020-09-24 at 14 10 00" src="https://user-images.githubusercontent.com/6961901/94143172-c2212b00-fe6f-11ea-90db-6080ea6957b5.png">

Reply:
you can even render different layout for large and small screens and you don't have to rely just on media queries. There is useLayoutSize hook that will return a boolean if the mobile layout is active 
` const { isMobileLayout } = useLayoutSize();`. So you can conditionally render different component (or just pass some prop to it) to do necessary adjustments